sql/moz-fx-data-shared-prod/mozilla_vpn_external/survey_product_quality_v1/query.py

```diff
@@ -0,0 +1,4 @@
+from bigquery_etl.alchemer.survey import main
+
+if __name__ == "__main__":
+    main()
```

Comment:
I'm assuming this will be added later?

Reply:
This is actually implemented, I'm just reusing the same click command in each one of these jobs.

Reply:
oh I see, cool!